pipeline/run_pipeline.py

```diff
@@ -351,36 +369,29 @@ def select_and_save_direction(
     return pos, layer, direction
 
 
-def select_and_save_direction_2(
+def for_refusal_metric_plots(
```

Comment:
You can remove this function completely by moving the makedirs line inside `refusal_metric_plot`?